mixer/cmd/root.go

```diff
@@ -33,6 +33,13 @@ import (
 
 var configFile string
 
+// Adding function here instead of including all of helpers package
+func runCommand(cmdname string, args ...string) error {
```

Comment:
The benefit of the wrapper is that you know all the places that use it are calling the underlying command in a uniform way. But if this is now a possibly-out-of-sync copy of that wrapper, its utility is lost.

Just use `exec.Command(...).Run()` directly below rather than copying this pretty thin wrapper.